rotkehlchen/tests/api/test_users.py

```diff
@@ -354,6 +354,68 @@ def test_user_creation_with_already_loggedin_user(rotkehlchen_api_server, userna
     )
 
 
+# @pytest.mark.parametrize('start_with_logged_in_user', [True]) (NEEDED?)
+def test_user_password_change(rotkehlchen_api_server, username, db_password):
+    """
+    Test that changing a logged-in user's users password works successfully and that
+    common errors are handled.
+    """
+
+    # wrong username
+    data_wrong_user = {
+        'name': 'billybob',
+        'password': 'asdf',
+        'new_password': 'asdf',
+    }
+    response = requests.patch(api_url_for(rotkehlchen_api_server, "userpasswordchangeresource",
+                              name=username), json=data_wrong_user)
+    msg = (
+        f'Provided user "{data_wrong_user["name"]}" is not the logged in user'
+    )
+    assert_error_response(
+        response=response,
+        contained_in_msg=msg,
+        status_code=HTTPStatus.BAD_REQUEST,
+    )
+
+    # wrong password
+    data_wrong_pass = {
+        'name': username,
+        'password': 'asdf',  # should be ??? '123' (?)
```

Comment:
you have the password you start with in the `db_password` fixture. It's actually passed as an argument in the start of this tests.

Reply:
Yeah that was just a note to myself at the time, the db_password in the fixture is 123, will clean up :)